src/app/exemples/admin/configuration.ts.dist

```diff
@@ -1,6 +1,13 @@
-export default {
+let config: any =  {
   api: {
     baseUrl: 'http://dockerhost:60000',
     definitionUrl: 'https://www.w3.org/ns/hydra/core'
-  }
+  }/*,
```

Comment:
The better solution is to write a full documentation on how to install the project, using it in an existant app and configure it which is quite out of the scope of this already too big PR IMO.
Until this documentation doesn't exist, I thought that a commented configuration could show you how you can configure the app